kotlin/manage-operational-layers/src/main/java/com/esri/arcgisruntime/sample/manageoperationallayers/MainActivity.kt

```diff
@@ -0,0 +1,161 @@
+/*
+ * Copyright 2020 Esri
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *    http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ *
+ */
+
+package com.esri.arcgisruntime.sample.manageoperationallayers
+
+import android.os.Bundle
+import android.view.MotionEvent
+import android.view.View
+import androidx.appcompat.app.AppCompatActivity
+import androidx.coordinatorlayout.widget.CoordinatorLayout
+import androidx.recyclerview.widget.ItemTouchHelper
+import androidx.recyclerview.widget.LinearLayoutManager
+import com.esri.arcgisruntime.layers.ArcGISMapImageLayer
+import com.esri.arcgisruntime.layers.Layer
+import com.esri.arcgisruntime.mapping.ArcGISMap
+import com.esri.arcgisruntime.mapping.Basemap
+import com.esri.arcgisruntime.mapping.view.DefaultMapViewOnTouchListener
+import kotlinx.android.synthetic.main.activity_main.*
+
+
+class MainActivity : AppCompatActivity() {
+
+  private val removedLayers = mutableListOf<Layer>()
+
+  override fun onCreate(savedInstanceState: Bundle?) {
+    super.onCreate(savedInstanceState)
+    setContentView(R.layout.activity_main)
+
+    // create a map with a topographic basemap
+    val map = ArcGISMap(Basemap.Type.TOPOGRAPHIC, 34.056295, -117.195800, 4)
+
+    val imageLayerElevation =
+      ArcGISMapImageLayer("https://sampleserver5.arcgisonline.com/arcgis/rest/services/Elevation/WorldElevations/MapServer")
+    val imageLayerCensus =
+      ArcGISMapImageLayer("https://sampleserver5.arcgisonline.com/arcgis/rest/services/Census/MapServer")
+    val imageLayerDamage =
+      ArcGISMapImageLayer("https://sampleserver5.arcgisonline.com/arcgis/rest/services/DamageAssessment/MapServer")
+
+    // add the map image layers to the map's operational layers
+    map.operationalLayers.addAll(listOf(imageLayerElevation, imageLayerCensus, imageLayerDamage))
+
+    // set the map to the map view
+    mapView.map = map
```

Comment:
```suggestion
    val imageLayerElevation =
    ArcGISMapImageLayer("https://sampleserver5.arcgisonline.com/arcgis/rest/services/Elevation/WorldElevations/MapServer")
    val imageLayerCensus =
      ArcGISMapImageLayer("https://sampleserver5.arcgisonline.com/arcgis/rest/services/Census/MapServer")
    val imageLayerDamage =
      ArcGISMapImageLayer("https://sampleserver5.arcgisonline.com/arcgis/rest/services/DamageAssessment/MapServer")

    // create a map with a topographic basemap and set it to the map view
    mapView.map = ArcGISMap(Basemap.Type.TOPOGRAPHIC, 34.056295, -117.195800, 4).apply {
      // add the map image layers to the map's operational layers
      operationalLayers.addAll(listOf(imageLayerElevation, imageLayerCensus, imageLayerDamage))
    }
```

Maybe I'm going overboard with inlining?

Reply:
Hm, I'm torn, this one is reasonable but it also seems like it's a lot of operations. Maybe we should try to define when we inline stuff in general? I can go either way here